### PR TITLE
[ANTORA] Improvements, batch 1

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

**Review by Arsnael — COMMENTED** [review]
LGTM.

Mind though that we might need to readapt, as modifications on that section are made here: https://github.com/apache/james-project/pull/2332

EDIT: wrong comment, was supposed to end in https://github.com/apache/james-project/pull/2334 so dont pay any attention to it

File: docs/modules/customization/pages/index.adoc [hunk 1]
```diff
@@ -1,3 +1,187 @@
-= Apache James Customization
-:navtitle: Customization
+= Extending server behavior
+:navtitle: Extending server behavior
 
+== Available extension mechanisms
+
+image::james-hexagons-extensions.png[Extension mechanisms for the Distributed Server]
+
+The Distributed Server exposes several interfaces allowing the user to write custom extensions in
```
Comment (Arsnael) [inline docs/modules/customization/pages/index.adoc:8]:
```suggestion
The James Server exposes several interfaces allowing the user to write custom extensions in
```

File: docs/modules/customization/pages/index.adoc [hunk 2]
```diff
@@ -1,3 +1,187 @@
-= Apache James Customization
-:navtitle: Customization
+= Extending server behavior
+:navtitle: Extending server behavior
 
+== Available extension mechanisms
+
+image::james-hexagons-extensions.png[Extension mechanisms for the Distributed Server]
```
Comment (Arsnael) [inline docs/modules/customization/pages/index.adoc:6]:
```suggestion
image::james-hexagons-extensions.png[Extension mechanisms for the James Server]
```

File: docs/modules/customization/pages/index.adoc [hunk 3]
```diff
@@ -1,3 +1,187 @@
...
+
+The Distributed Server exposes several interfaces allowing the user to write custom extensions in
+order to extend the Distributed Server behavior.
```
Comment (Arsnael) [inline docs/modules/customization/pages/index.adoc:9]:
```suggestion
order to extend the James Server behavior.
```

File: docs/modules/development/pages/index.adoc [hunk 4]
```diff
@@ -1,5 +1,10 @@
 = Apache James Developer Guide
 :navtitle: Developer Guide
 
-(TODO)
+This page aggregates common knowlege for contributing to APache james servers.
```
Comment (Arsnael) [inline docs/modules/development/pages/index.adoc:4]:
```suggestion
This page aggregates common knowlege for contributing to Apache james servers.
```

**Review by Arsnael — APPROVED** [review]

**Review by vttranlina — COMMENTED** [review]
read it,

**Review by quantranhong1999 — APPROVED** [review]

**Comment by Arsnael** [comment]
Squashed the fixup

**Pull request merged**